### PR TITLE
Display group page lists (members and products)

### PR DESCRIPTION
**Pull request by alternadiva**

### CODE REVIEW

File: components/Groups/ListGroups.js [hunk 1]
```diff
@@ -11,7 +13,7 @@ function ListGroups(props) {
         {props.groups.map((group) => {
           return (
             <li
-              key={group.id}
+              key={RandomKey()}
```
Comment (sonianb) [inline components/Groups/ListGroups.js:16]: Didn't know about `RandomKey()`. Nice!

**Review by minju25kim — APPROVED** [review]

File: components/Hooks/RandomKey.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function RandomKey() {
```
Comment (sonianb) [inline components/Hooks/RandomKey.js:1]: Good idea!

File: components/Products/ProductUpload.js [hunk 3]
```diff
@@ -61,8 +60,8 @@ function ProductUpload(props) {
       console.log(error.message);
     }
 
-    // if user click submit then it will be redirected to product page
-    router.push(`/products/${data[0].id}`);
+    // if user click submit then it will be redirected to group page
+    router.push(`/products/${groupId}`);
```
Comment (sonianb) [inline components/Products/ProductUpload.js:64]: Nice fix!

**Review by sonianb — APPROVED** [review]

**Comment by sonianb** [comment]
[Select and display members from the database.](https://trello.com/c/TR1U8KuN/37-select-and-display-members-from-the-database)

**Comment by sonianb** [comment]
[When user creates a group, they become members of the group.](https://trello.com/c/emVi26fp/67-when-user-creates-a-group-they-become-members-of-the-group)

**Review by AsmahanM — APPROVED** [review]

**Comment by sonianb** [comment]
[Select and display items from the database on Group pages](https://trello.com/c/p81auZV4/35-select-and-display-items-from-the-database-on-group-pages)

**Pull request merged**